cli/githubutil.py

```diff
@@ -82,4 +86,47 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def remove_outdated_webhooks(
+    github_org_name: str,
+    github_cfg_name: str,
+    concourse_cfg_name: str,
+):
+    '''
+    Removes webhooks with query parameter 'concourse_id' whose value does not match
+    parameter 'concourse_cfg_name'. Other webhooks are not processed
+    '''
+    cfg_factory = ctx().cfg_factory()
+    github_cfg = cfg_factory.github(github_cfg_name)
+
+    github_api = _create_github_api_object(github_cfg=github_cfg)
+    github_org = github_api.organization(github_org_name)
+
+    for repository in github_org.repositories():
+        print("")
+        print("Processing repository: {}".format(repository.name))
+        repo = github_api.repository(
+            owner=github_org_name,
+            repository=repository.name
+        )
+        try:
+            for hook in repo.hooks():
```

Comment:
this pretty much duplicates the logic contained in github/__init__.py#GithubWebHookSyncer

--> why not use instead:

github_webhook_syncer(github=github_api)
def filter_fun(url):
  # pseudo.code:
  if "url contains query-parameter concourse_id= our id":
   return True # instruct syncer to rm hook

github_webhook_syncer.remove_outdated_hooks(owner=owner, repository_name=repository_name, urls_to_keep=(,), url_filter_fun=filter_fun)